lib/scope_file_access.rb

```diff
@@ -30,6 +30,16 @@ def file_content(filename)
 
     File.read(path)
   end
+
+  def binary?(filename)
```

Comment:
Shouldn't the method be the other way around and check, if a file is a text file? That is what is actually doing, so naming it `text_file?` might be more accurate.